indico/modules/events/contributions/models/contributions.py

```diff
@@ -319,6 +332,12 @@ def get_non_inheriting_objects(self):
         """Get a set of child objects that do not inherit protection"""
         return get_non_inheriting_objects(self)
 
+    def get_field_value(self, field_id):
+        field_ids = {v.contribution_field.id: v.value for v in self.field_values}
+        if field_id in field_ids:
```

Comment:
`return field_ids.get(field_id, '')` - also, `field_ids` is not a good name since the dict contains the field values. also, I'd use `v.contribution_field_id` in the dict comprehension (no need to trigger a load on the relationship just to get the ID)